src/Sulu/Bundle/CategoryBundle/Tests/Unit/Content/Types/CategoryListTest.php

```diff
@@ -58,7 +58,7 @@ public function testGetContentDataNullPropertyValue()
         $categoryManager = $this->prophesize(CategoryManagerInterface::class);
         $categoryManager->findByIds(Argument::any())->shouldNotBeCalled();
 
-        $categoryList = new CategoryList($categoryManager->reveal(), '');
+        $categoryList = new CategorySelection($categoryManager->reveal(), '');
```

Comment:
This file should also be renamed to `CategorySelectionTest`, right?

Reply:
And btw, isn't this file in the wrong namespace?

Reply:
@danrot 👍